Gulpfile.mjs

```diff
@@ -636,14 +636,16 @@ function buildRollupDts(packages) {
     await build(input, output, "", packageName);
   });
 
-  tasks.push(
-    build(
-      "packages/babel-parser/typings/babel-parser.source.d.ts",
-      "packages/babel-parser/typings/babel-parser.d.ts",
-      "// This file is auto-generated! Do not modify it directly.\n/* eslint-disable @typescript-eslint/consistent-type-imports, prettier/prettier */",
-      "packages/babel-parser"
-    )
-  );
+  if (bool(process.env.BABEL_8_BREAKING)) {
+    tasks.push(
```

Comment:
Why only for Babel 8? We already publish this file for Babel 7 too, as far as I know.

Reply:
In this PR we removed some plugins, which seems to be a breaking change for TS users.

Reply:
Removing the `.d.ts` file completely is also a breaking change, right?

Reply:
No, it just can't be generated automatically anymore. :)
The ones we edited manually are still there.

Reply:
Right, but the one we publish is the auto-generated one: https://yarnpkg.com/package?q=%40babel%2Fparser&name=%40babel%2Fparser&file=%2Ftypings%2Fbabel-parser.d.ts